src/components/CreateCoursePage/index.jsx

```diff
@@ -24,12 +24,13 @@ class CreateCoursePage extends React.Component {
       number: options.number,
       mode: options.enrollmentTrack,
       price: options.price,
+      course_run: {
+        course: `${options.org}+${options.number}`,
```

Comment:
Why are we constructing this here? I highly recommend we move this to the backend.  And avoid constructing keys on the frontend as much as possible.

Reply:
We could also use the key that is generated when the course is generated on the backend.

Reply:
Yeah, it rubbed me the wrong way too. Will update.